charts/metallb/templates/speaker.yaml

```diff
@@ -447,7 +453,7 @@ spec:
         args:
           - --logtostderr
           - --secure-listen-address=:{{ .Values.prometheus.secureMetricsPort }}
-          - --upstream=http://$(METALLB_HOST):{{ .Values.prometheus.metricsPort }}/
+          - --upstream=http://127.0.0.1:{{ .Values.prometheus.metricsPort }}/
```

Comment:
Does it work in a ipv6-only cluster?

Reply:
this is a very valid point :sweat_smile: